input/fsh/SD_Extensions.fsh

```diff
@@ -181,6 +181,15 @@ Description: "Radiotherapy Treatment Applicator Type."
 * value[x] only CodeableConcept
 * value[x] from ApplicatorTypesVS (extensible)
 
+Extension: UniformFractionation
+Id: codexrt-radiotheraphy-uniform-fractionation
+Title: "Uniform Fractionation"
+Description: "Uniform Fraction is true if fractionation was unform, false if fractionation was mixed."
```

Comment:
typo: "unform" Maybe add more text that "uniform" here means with same modality, technique, and dose per fraction?

Reply:
looks like you added it...